editor/src/components/canvas/canvas-strategies/strategies/reorder-utils.ts

```diff
@@ -51,7 +51,7 @@ export function applyReorderCommon(
     const selectedElements = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
     const target = selectedElements[0]
 
-    const siblings = MetadataUtils.getSiblingsUnordered(canvasState.startingMetadata, target).map(
+    const siblings = MetadataUtils.getSiblingsOrdered(canvasState.startingMetadata, target).map(
```

Comment:
of course we need _the ordered_ siblings here, since we are actually trying to determine a reorder index!!!